src/app/ui-kit/button/button.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html',
+})
+export class ButtonComponent implements OnInit {
+  @Input() type: 'accent' | 'ghost' = 'accent';
+  @Input() size: 'sm' | 'md' = 'sm';
+  @Input() iconState: string = '';
+  @Input() disabled = false;
```

Comment:
if you chose to set the type of a value, let's do that for all cases: disabled: boolean = false;